lib/cookie.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+
+ // needed to communicate/translate between JSONWire cookies and regular JS cookies 
+
+import _ from 'lodash';
+import { getLogger } from 'appium-logger';
+
+
+export default class Cookie {
+
+  constructor (cookieString) {
+    this.cookieString = cookieString;
+    this.log = getLogger('mymodule');
+  }
+ 
+  // parses the value if needed and converts the value if a converter is provided
+  convertCookie (value, converter) {
+    let parsedValue;
+
+    if (value.indexOf('"') === 0) {
```

Comment:
Actually... the original comment is useful: https://github.com/carhartl/jquery-cookie/blob/master/src/jquery.cookie.js#L37